engine/src/main/java/org/terasology/rendering/dag/nodes/FinalPostProcessingNode.java

```diff
@@ -203,6 +187,6 @@ public void propertyChange(PropertyChangeEvent event) {
             }
         } // else: no other cases are possible - see subscribe operations in initialize().
 
-        worldRenderer.requestTaskListRefresh();
+        // worldRenderer.requestTaskListRefresh();
```

Comment:
Why did you comment this out? I would have thought it necessary after a change in the desired states list, whether it is an addition or a removal.